pkgs/servers/pleroma/mix.nix

```diff
@@ -281,6 +281,19 @@ let
       beamDeps = [ ecto ];
     };
 
+    crypt = buildRebar3 rec {
+      name = "crypt";
+      version = "1.0.1";
+
+      src = fetchHex {
+        pkg = "${name}";
```

Comment:
```suggestion
        pkg = pname;
```

Reply:
Is it automatically generated? If it is, then, I think something else should change.